app/javascript/articles/__tests__/Article.test.jsx

```diff
@@ -263,4 +263,25 @@ describe('<Article /> component', () => {
 
     expect(queryByText('person', { selector: 'span' })).toBeDefined();
   });
+
```

Comment:
@aitchiss Here are the specific tests for the presence or absence of the Bookmark Button on articles, FYI.

Reply:
Love it!